sdk/core/core-amqp/src/ConnectionContextBase.ts

```diff
@@ -144,13 +144,14 @@ export const ConnectionContextBase = {
       (!isNode && typeof window !== "undefined" && (window as any).WebSocket)
     ) {
       const socket = parameters.config.webSocket || (window as any).WebSocket;
-      const host = parameters.config.host;
+      const host = connectionOptions.host;
       const endpoint = parameters.config.webSocketEndpointPath || "";
       const socketOptions = parameters.config.webSocketConstructorOptions || {};
+      const port = parameters.config.port ?? 443;
```

Comment:
My assumption was that when we use web sockets, we would always use port 443 and port customization was only relevant to non web socket scenario. Is that not the case?

Reply:
I was able to configure an application gateway front-end that forwarded a different port to the event hubs (ultimately to 443 on the backend), so it can definitely be something other than 443.